universal_transfer.md

```diff
@@ -0,0 +1,112 @@
+# Universal Transfer Method for Open-Attestation Documents
+
+## Status
+
+Draft
+
+## Rationale
+
+The goal of the universal transfer method is to ensure that a standard is available to all clients interacting with OA documents. This allows different clients to be itneropable with one another. The standard is extremely important if we want to provide a client like an [identity wallet](https://github.com/Open-Attestation/identity-wallet) that supports all kinds of OA document.
+
+Important considerations of the transfer method:
+
+- Reference to resource (uri)
+- Encrypted resource (key)
+- Permitted actions (termsOfUse)
+- Safe transfer (encoding)
+- Universal resolver (deep-linking)
+
+## Proposed Solution
+
+Sample QR code:
+
+![Proposed QR](assets/universal_transfer/proposed-qr.png)
+
+```
+https://openattestation.com/?resource=%7B%22uri%22%3A%22https%3A%2F%2Fsomehostedresources.com%2Fdoc%2F7de3bce4-de62-4628-914e-97d41e642582%22%2C%22enc%22%3A%7B%22type%22%3A%22OPEN-ATTESTATION-TYPE-1%22%2C%22key%22%3A%22aa57eb519fd3c63c42c2f2697e8957198b56fc945c4db18b480c07d2e6485a93%22%7D%2C%22permittedAction%22%3A%5B%22VIEW%22%2C%22STORE%22%5D%7D
+```
+
+Decoded Resource (after `?resource=`):
+
+```json
+{
+  "uri": "https://somehostedresources.com/doc/7de3bce4-de62-4628-914e-97d41e642582",
+  "enc": {
+    "type": "OPEN-ATTESTATION-TYPE-1",
+    "key": "aa57eb519fd3c63c42c2f2697e8957198b56fc945c4db18b480c07d2e6485a93"
+  },
+  "permittedAction": ["VIEW", "STORE"],
```

Comment:
As the holder of the document, can I change these actions as I please? Or will there be a scenario where I'm restricted from performing certain actions.

I'm likening this to the idea of Terms of Use (as you mentioned later on) where the issuer could impose restrictions to the subsequent actions that the holder can do.

Reply:
As the holder of a document (whether you are the subject or not) you can do as you please. This field is added to represent intent or `termsOfUse` in w3c vc lingo. 

Kind of like I gave you a scan of my NRIC and told you not to share with anyone else. But whether you respect that it's on you. You might be subjected to some sort of legal liability otherwise. 

In W3C VC data model, they made sure to have the holder sign the `presentation`. Had a discussion with RJ yesterday this where we both agree that following their presentation model might be a lot cleaner but the interim implementation can look like that.